cmake/helpers/GdalGenerateConfig.cmake

```
@@ -1,3 +1,23 @@
+if (NOT WINDOWS OR MINGW)
```

It doesn't look like WINDOWS is a CMake variable, and when looking at that line, I had a second of doubt about operator priority, so I'd suggest:
```suggestion
if ((NOT WIN32) OR MINGW)
```

---

You are right about `WIN32`. This explains why I didn't see the left-over message output... I really need a few days of holidays ;-)
But there is no need to worry about operator precedence here, it is okay without extra markup.

---

Done.